snapcraft/plugins/make.py

```diff
@@ -42,6 +42,9 @@ def schema(cls):
         schema['properties']['makefile'] = {
             'type': 'string'
         }
+        schema['properties']['make-parameters'] = {
```

Comment:
You should add `make-parameters` to the docstring above as well.

Btw. Is there a specific use for this?